### DISCUSSION TITLE
Preserve compose file blank lines

### DISCUSSION THREAD

**Discussion by odwulf** [louislam/dockge#418] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

Dockge does a great job at handling the existing compose files without messing them up by moving things around. I just would love for that care to extend to the blank lines I insert for legibility.

### ✔️ Solution

If a blank line exists, keep it.

### 📝 Additional Context

Not that I expect Dokge to guess my patterns and use them for new containers, but the big plus of the app compared to other solutions is the ability to work with existing stacks and compose files while keeping them intact enough to keep my others workflows, and (for me) seeing my files become a wall of text kinda defeats that purpose.

**Comment by richard-lee-863** [comment]
I second this, I find it quite hard to read some of the larger compose stacks when the blank lines are removed..
